docs/development-environment.md

```diff
@@ -159,29 +163,31 @@ cd jetpack
 
 ## Building Jetpack
 
-To work on Jetpack you need to build the JavaScript and CSS components of the plugin's admin interface. This will generate the run time bundle (`_inc/build/admin.js`)
+To work on Jetpack you need to build PHP, JavaScript, and CSS components of the packages and the plugins' interfaces.
+
+[The Jetpack CLI tool](https://github.com/Automattic/jetpack/tree/master/tools/cli/README.md) will help you with all building steps.
```

Comment:
```suggestion
[The Jetpack CLI tool](https://github.com/Automattic/jetpack/blob/master/tools/cli/README.md) will help you with all building steps.
```
I think we need `blob` here instead of `tree`?